setup.py

```diff
@@ -14,14 +14,12 @@
 #    License for the specific language governing permissions and limitations
 #    under the License.
 
-import os
-
 from setuptools import setup
 from setuptools import find_packages
 from pip.req import parse_requirements
 
 setup(name='CloudFerry',
-      version=os.environ['CLOUDFERRY_VERSION'],
+      version=open('version.txt').read().strip(),
```

Comment:
In this case file remains open. I guess it would be better to close it and free up any system resources taken up by the open file (https://docs.python.org/2/tutorial/inputoutput.html).